support-frontend/assets/pages/paper-subscription-landing/paperSubscriptionLandingPage.jsx

```diff
@@ -48,10 +49,12 @@ const content = (
     >
       <CampaignHeader />
       {paperHasDeliveryEnabled() &&
-        <Content needsHigherZindex>
+        <Content needsHigherZindex innerBackground="grey">
           <Text>
             <LargeParagraph>
-              We offer two different subscription types: voucher booklets and home delivery.
+              {useDigitalVoucher === null || useDigitalVoucher === false || useDigitalVoucher === undefined
```

Comment:
can `useDigitalVoucher` ever be undefined? Doesn't it default to null?

Reply:
Yes it does (default to null). At one point it was undefined but I think that was due to another issue I was having so most likely overkill! I will remove.

Reply:
`!useDigitalVoucher` seemed to cover it all off!